### DISCUSSION TITLE
I have a problem for a long time

### DISCUSSION THREAD

**Discussion by sumukusu** [airsquared/blobsaver#284] [Q&A] [answered]
hello, i saved my iOS 14.3 shsh with blobssaver2 (the buggy version) and i can't see the "generator" code to downgrade my ios version with futurerestore, can you see the file and give me the generator code? please i need it and i have been waiting months to download ios 14.3 with futurerestore, i need your help please
shsh: https://drive.google.com/file/d/1QRhIVE7K9TDNzz_5zKpem4Ldp_fh0B8h/view

**Comment by airsquared** [comment, marked as answer]
Were you jailbroken or did you have a generator/nonce set when you saved blobs?

**Reply by sumukusu** [reply]
i saved the blobs with jailbroken iphone and i don't remember if i set the nonce or not

**Reply by airsquared** [reply]
If you were jailbroken, then your nonce was already set by the jailbreak you were using. Coolstar's jailbreaks use something like`0xbd3...` and unc0ver uses something like `0x111...`.

**Reply by sumukusu** [reply]
i tried to use 0x111.. in futurerestore and dont work. when i saved the 14.3 shsh i used checkra1n jailbreak, now too

**Reply by airsquared** [reply]
It is a possibility that your nonce/generator was not set by checkra1n when you read the APNonce from your device which is why the APNonce doesn't match now.

**Reply by sumukusu** [reply]
so my shsh of 14.3 will never work? :c

**Reply by airsquared** [reply]
If neither of those generators work and you didn't set your own generator when reading the APNonce, sorry looks like there isn't much you can do.

**Reply by sumukusu** [reply]
anyway, thanks for the help :c

**Reply by r1sk01** [reply]
ow, that really sucks @bugmen6 :(
